packages/transaction-controller/package.json

```diff
@@ -92,7 +92,7 @@
   },
   "peerDependencies": {
     "@babel/runtime": "^7.23.9",
-    "@metamask/accounts-controller": "^18.0.0",
+    "@metamask/accounts-controller": "^19.0.0",
```

Comment:
Hmm, looks like we also need to include `transaction-controller` in the release now.

The good news is that the only package that seems to have a peer dep on `transaction-controller` is `user-operation-controller`, which we are already releasing, so I don't think we need to release anything else beyond this. We would just need to bump the peer dep in `user-operation-controller`.